pwa-devdocs/src/technologies/basic-concepts/container-extensibility/index.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: Container extensibility
+adobeio: /guides/general-concepts/container-extensibility/
```

Comment:
This topic is no longer relevant and doesn't need to be ported over. It hasn't been listed in the nav for a long time and I probably should have archived it.